addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -488,6 +488,23 @@ img::selection {
     }
 }
 
+.transfo-container {
+    .transfo-controls {
+        .transfo-rotator {
+            margin: 0 !important;
+            opacity: 0.5;
+
+            &:hover {
+                opacity: 1;
+            }
+
+            &.active {
+                opacity: 1;
+            }
+        }
```

Comment:
&:hover, &.active {
                opacity: 1;
            }
I'd rather do this to save space 😉